test/es-module/test-esm-loader-http-imports.mjs

```diff
@@ -53,20 +85,14 @@ const {
     ]
   );
 
-  let stderr = '';
-  let stdout = '';
-
-  child.stderr.setEncoding('utf8');
-  child.stderr.on('data', (data) => stderr += data);
-  child.stdout.setEncoding('utf8');
-  child.stdout.on('data', (data) => stdout += data);
+  const stderr = (await child.stderr.toArray()).join('');
+  const stdout = (await child.stdout.toArray()).join('');
+  const [code, signal] = await once(child, 'close');
 
-  child.on('close', mustCall((code, signal) => {
-    strictEqual(stderr, '');
-    strictEqual(stdout, '[Module: null prototype] { sum: [Function: sum] }\n');
-    strictEqual(code, 0);
-    strictEqual(signal, null);
+  strictEqual(stderr, '');
+  strictEqual(stdout, '[Module: null prototype] { sum: [Function: sum] }\n');
+  strictEqual(code, 0);
+  strictEqual(signal, null);
 
-    server.close();
-  }));
+  server.close();
```

Comment:
This change looks unrelated.

Reply:
it is unrelated, but since I have a single HTTP server I need to close after two tests run, it made more sense to move this test as well to work with top-level await